packages/plugin-core/src/commands/SetupWorkspace.ts

```diff
@@ -85,39 +162,35 @@ export class SetupWorkspaceCommand extends BasicCommand<
     skipConfirmation?: boolean;
   }): Promise<boolean> => {
     if (fs.existsSync(rootDir) && !skipConfirmation) {
-      const options = {
-        delete: { msg: "delete existing folder", alias: "d" },
-        abort: { msg: "abort current operation", alias: "a" },
-        continue: {
-          msg: "initialize workspace into current folder",
-          alias: "c",
-        },
-      };
-      const resp = await vscode.window.showInputBox({
-        prompt: `${rootDir} exists. Please specify the next action. Your options: ${_.map(
-          options,
-          (v, k) => {
-            return `(${k}: ${v.msg})`;
-          }
-        ).join(", ")}`,
-        ignoreFocusOut: true,
-        value: "continue",
-        validateInput: async (value: string) => {
-          if (!_.includes(_.keys(options), value.toLowerCase())) {
-            return `not valid input. valid inputs: ${_.keys(options).join(
-              ", "
-            )}`;
-          }
-          return null;
-        },
-      });
+      const resp = await VSCodeUtils.showQuickPick(
```

Comment:
kudos for including this. this has been on my backlog since the moment it was implemented 😅